packages/client/src/editor/components/StickyNote.tsx

```diff
@@ -43,28 +28,27 @@ const StickyNote: React.FunctionComponent<NoteProps> = props => {
   };
 
   return (
-    <div 
-      ref={ref} 
-      style={style} 
-      className="stickyNote"
-      onMouseMove={ (e) => onMouseMove(e) }
-      onMouseDown={ () => setMouseDown(true) }
-      onMouseUp={ () => setMouseDown(false) }>
-      <textarea
-        onChange={e => props.save(note.id, e.target.value)}
-        value={note.text}
-      />
-      <span
-        className="delete"
-        title="Delete"
-        onClick={e => {
-          e.preventDefault();
-          deleteNote();
-        }}
-      >
-        <DeleteIcon />
-      </span>
-    </div>
+    <Draggable 
```

Comment:
The sticky-note does not get pushed around with the react-draggable component.

The websocket is perpetually in connecting state so could not test that note moves are correctly updated across clients...